### ISSUE TITLE
[Suggestion] Loading torch-model-archiver arguments from YAML

### ISSUE THREAD

**Issue by skat00sh** [pytorch/serve#780]
## Is your feature request related to a problem? Please describe.

`torch-model-archiver` takes in a lot of params that makes it a hefty command line argument
## Describe the solution

Allowing all the param values to be parsed from a YAML file when `torch-model-archiver` is used with a `--yaml` flag.
It will also help automating creation of multiple `.mar` from different configurations.
## Describe alternatives solution

I've tried calling `torch-model-archiver` from a shell script with variable `params` and that works file as well. But I just wanted to bounce this idea off the community to see if it is something people could find useful. 
I'd be happy to work on it.

**Comment by skat00sh** [comment]
@dhaniram-kshirsagar @harshbafna Any thoughts on this?

**Comment by dhaniram-kshirsagar** [comment]
@skat00sh Yes, it will be nice to have YAML/json as an alternative to command input.

**Comment by skat00sh** [comment]
Thanks @dhaniram-kshirsagar! Will start working on it soon

**Comment by msaroufim** [comment]
Hi @skat00sh let me know if you're still interested in working on this so we get something merged. This involves 4 steps

1. Python has a nice YAML library https://stackabuse.com/reading-and-writing-yaml-to-a-file-in-python/
2. And would need to update the arg-parser https://github.com/pytorch/serve/blob/master/model-archiver/model_archiver/arg_parser.py
3. Adding a sample YAML config in this directory https://github.com/pytorch/serve/tree/master/model-archiver
4. Testing the model with a YAML config and validating that it runs

**Comment by skat00sh** [comment]
@msaroufim Sorry, I almost forgot about this issue! I'm back on it  now, will finish it asap! Thanks for defining the steps!

**Comment by skat00sh** [comment]
@msaroufim I think I should release it for now for someone else. With the academic workload this sem, I'm finding it a bit hard to find time for anything. Sorry that I took so long again to revert.

I wished to mention 2 more things though:
1. Is there a chat-group or discord server like thing that I could join to be a part of the development in the near future? I find the revert latency at github issues sometimes is a bit too much (I'm guilty of it myself)
2. While trying to setup the dev environment, I faced a couple of issues. I unfortunately didn't log them but if I'm working on it again, what do you think should be the quickest way to reach out to devs?
